### PR TITLE
move rst_epilog subs and links to file

### PR DESCRIPTION
**Pull request by nstarman**
### Description

@embray @mhvk, as suggested. I agree this looks better.

@pllim : no-changelog-needed ? v4.3 ? Thanks!

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -109,109 +109,9 @@
...
-
-.. py.test
-.. _pytest: https://pytest.org/en/latest/index.html
-.. _pytest-astropy: https://github.com/astropy/pytest-astropy
-.. _pytest-doctestplus: https://github.com/astropy/pytest-doctestplus
-.. _pytest-openfiles: https://github.com/astropy/pytest-openfiles
-.. _pytest-remotedata: https://github.com/astropy/pytest-remotedata
-
-"""
+with open("common_links.txt", "w") as cl:
```
Comment (adrn) [inline docs/conf.py:112]: Docs failure is real, I think because:
```suggestion
with open("common_links.txt", "r") as cl:
```

**Review by adrn — COMMENTED** [review]
Great idea! Small typo here though (see comment)

File: docs/conf.py (hunk 1 above)
Comment (nstarman) [reply docs/conf.py:112]: Lol. thanks! I opened this PR and moved to the next thing before ``tox -e build_docs`` has started up properly... so I think it tested my other branch (which passed TYVM)!

Comment (nstarman) [reply docs/conf.py]: Is there any way to speed up ``tox -e build_docs`` on the ``build_docs inst-nodeps: [path]/local/astropy/.tox/.tmp/package/1/astropy-4.3.dev1188+gf5883030e.tar.gz`` initial stage. It's sooo slow. Like cache some stuff?

Comment (adrn) [reply docs/conf.py]: Oof, not that I know if, but if you figure out a way I would be keen to know! Related to #10470 I supposed..

**Comment by adrn** [comment]
(BTW: you could have added `[ci skip]` to your commit message, which cancels all GH actions tests but still runs the circleci/readthedocs tests.)

**Comment by pllim** [comment]
Re: skip ci -- CircleCI theoretically would get skipped too but I notice that they don't always. 🤷  RTD does not skip.

**Review by adrn — APPROVED** [review]
Thanks for this follow-up!

**Comment by mhvk** [comment]
Nice! Perhaps should have been a `.rst` filetype, but not a big deal.

**Comment by nstarman** [comment]
We actually can't use this in the docs or even transclude this since ``rst_epilog`` would attach it to itself and error from duplicate definitions! I think .txt obviates this and the orphan rst file problem.

**Comment by mhvk** [comment]
Ah, makes sense, thanks!

**Comment by embray** [comment]
Thanks for the quick turnaround on this!  :+1:

**Pull request merged**